README.md

```diff
@@ -39,7 +43,7 @@ cd ~/MagicMirror/modules
 Clone this repository:
 
 ```bash
-git clone https://github.com/ianperrin/MMM-Formula1
+git clone https://github.com/73cirdan/MMM-Formula1
```

Comment:
This should be undone before the PR is accepted.

Reply:
updated package.json to ianperrin